arches/app/datatypes/datatypes.py

```diff
@@ -320,10 +321,14 @@ def transform_value_for_tile(self, value, **kwargs):
             value = value[0]
         valid_date_format, valid = self.get_valid_date_format(value)
         if valid:
-            value = datetime.strptime(value, valid_date_format).astimezone().isoformat(timespec="milliseconds")
+            v = datetime.strptime(value, valid_date_format)
         else:
             v = datetime.strptime(value, settings.DATE_IMPORT_EXPORT_FORMAT)
-            value = v.astimezone().isoformat(timespec="milliseconds")
+        if (not sys.platform.startswith("win")) or (
+            v >= datetime.fromtimestamp(0)
+        ):  # The .astimezone() function throws an error on Windows for dates before 1970
+            v = v.astimezone()
```

Comment:
Unfortunately we can't skip astimezone because it is necessary to properly format the tilevalue to ensure that the value works properly with search. Also, depending on how the incoming data is formatted, the data will fail without calling astimezone. Instead of skipping astimezone, what is needed is equivalent logic.